### PR TITLE
Tam

### PR DESCRIPTION
**Pull request by MinhTam1509**
create method resovleFriendRequest, unFollow

### CODE REVIEW

File: src/main/java/com/cg_vibely_social_service/controller/FriendController.java [hunk 1]
```diff
@@ -39,18 +40,24 @@ public ResponseEntity<?> getFriendList (@PathVariable("id") Long id ) {
...
+
+    @DeleteMapping("/requestResolve")
+    public ResponseEntity<?> resolveRequest(@RequestBody ResolveRequestDto resolveRequestDto) {
+        friendRequestService.resolveRequest(resolveRequestDto);
         return new ResponseEntity<>(HttpStatus.OK);
     }
 
     @GetMapping("/requested/{id}")
-    public ResponseEntity<?> getRequestedFriend (@PathVariable("id") Long id) {
+    public ResponseEntity<?> getRequestedFriend(@PathVariable("id") Long id) {
```
Comment (ChungNguyenVr912) [inline src/main/java/com/cg_vibely_social_service/controller/FriendController.java]: redundant pathvariable parameter, use UserImpl from userService instead

File: src/main/java/com/cg_vibely_social_service/controller/FriendController.java [hunk 2]
```diff
@@ -39,18 +40,24 @@ public ResponseEntity<?> getFriendList (@PathVariable("id") Long id ) {
...
         List<FriendRequestDto> friendRequestList = friendRequestService.findAllFriendRequestByUserId(id);
         return new ResponseEntity<>(friendRequestList, HttpStatus.OK);
     }
 
     @GetMapping("/request/{id}")
-    public ResponseEntity<?> getFriendRequest (@PathVariable("id") Long id) {
```
Comment (ChungNguyenVr912) [inline src/main/java/com/cg_vibely_social_service/controller/FriendController.java:53]: redundant

File: src/main/java/com/cg_vibely_social_service/payload/request/FollowRequestDto.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.cg_vibely_social_service.payload.request;
+
+import lombok.*;
```
Comment (ChungNguyenVr912) [inline src/main/java/com/cg_vibely_social_service/payload/request/FollowRequestDto.java]: plz use specifically import

File: src/main/java/com/cg_vibely_social_service/payload/request/ResolveRequestDto.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package com.cg_vibely_social_service.payload.request;
+
+import lombok.*;
```
Comment (ChungNguyenVr912) [inline src/main/java/com/cg_vibely_social_service/payload/request/ResolveRequestDto.java]: plz use specifically import

File: src/main/java/com/cg_vibely_social_service/payload/response/FollowResponseDto.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
+package com.cg_vibely_social_service.payload.response;
+
+
+import lombok.*;
```
Comment (ChungNguyenVr912) [inline src/main/java/com/cg_vibely_social_service/payload/response/FollowResponseDto.java]: plz use specifically import

**Review by ChungNguyenVr912 — CHANGES_REQUESTED** [review]

File: src/main/java/com/cg_vibely_social_service/controller/FriendController.java (hunk 1 above)
Comment (MinhTam1509) [reply src/main/java/com/cg_vibely_social_service/controller/FriendController.java]: resolved

File: src/main/java/com/cg_vibely_social_service/controller/FriendController.java (hunk 2 above)
Comment (MinhTam1509) [reply src/main/java/com/cg_vibely_social_service/controller/FriendController.java:53]: resolved

File: src/main/java/com/cg_vibely_social_service/payload/request/FollowRequestDto.java (hunk 3 above)
Comment (MinhTam1509) [reply src/main/java/com/cg_vibely_social_service/payload/request/FollowRequestDto.java]: resolved

File: src/main/java/com/cg_vibely_social_service/payload/request/ResolveRequestDto.java (hunk 4 above)
Comment (MinhTam1509) [reply src/main/java/com/cg_vibely_social_service/payload/request/ResolveRequestDto.java]: resolved

File: src/main/java/com/cg_vibely_social_service/payload/response/FollowResponseDto.java (hunk 5 above)
Comment (MinhTam1509) [reply src/main/java/com/cg_vibely_social_service/payload/response/FollowResponseDto.java]: resolved

**Pull request merged**